Add Libero infrastructure scope ADR (#33)
libero/community: doc/adr/0008-libero-infrastructure.md

# 8. Limit scope of Libero infrastructure

Date: 2020-01-10

## Status

Proposed

## Context

Libero infrastructure (servers, Kubernetes, buckets) supports Libero development by providing demo environments.

Service providers need documentation to learn to run Libero products.

Service providers cater for the disparate, very specific needs of publishers.

Service providers may consolidate their infrastructure with the rest of the platforms.

## Decision

Libero infrastructure should serve two purposes:

- provide *demo* environments to showcase Libero products in certain configurations

> - provide realistic *reference* environments that do not serve real users but can be forked and adapted by service providers to kick start their Libero offering

Reviewer: Take 'demo' to be something that _is_ running, and 'reference' to be something that is _not_ running.

Author: Would only deploy `reference` to make sure it's working, so could be on each build of whatever Infrastructure As Code repository represents it. At the start it probably coincides with `demo` but can see that they may diverge later.

## Consequences

> Vanilla configuration of Libero products should always be running on Libero infrastructure.

Reviewer: ```suggestion
Demo environments for Libero products are run on Libero infrastructure.
```
